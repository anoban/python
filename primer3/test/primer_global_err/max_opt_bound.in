SEQUENCE_ID=max_opt_bound
SEQUENCE_TEMPLATE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_MIN_BOUND=94.0
PRIMER_OPT_BOUND=97.0
PRIMER_MAX_BOUND=96.0
=
