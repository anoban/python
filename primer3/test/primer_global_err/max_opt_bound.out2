../src/primer3_core: Optimum primer fraction binding lower than minimum or higher than maximum
