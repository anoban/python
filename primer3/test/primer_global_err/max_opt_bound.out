SEQUENCE_ID=max_opt_bound
SEQUENCE_TEMPLATE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_MIN_BOUND=94.0
PRIMER_OPT_BOUND=97.0
PRIMER_MAX_BOUND=96.0
PRIMER_ERROR=Optimum primer fraction binding lower than minimum or higher than maximum
=
